api/modules/os/path.py

```diff
@@ -0,0 +1,3 @@
+def realpath(path: str) -> str:
```

Comment:
wait...do you need to regenerate the html?

Reply:
That should happen when you run `tilt up`, right?